cwf/gateway/helm/cwf/templates/bin/_install_gateway.sh.tpl

```diff
@@ -162,14 +159,27 @@ cp .env /var/opt/magma/docker/
 cp recreate_services.sh /var/opt/magma/docker/
 cp recreate_services_cron /etc/cron.d/
 
+# When DPI is enabled
+if [ "$GW_TYPE" == "$CWAG" ] && [ -f "$DPI_LICENSE_NAME" ]; then
+  MODULE_DIR="cwf"
+  mkdir -p /var/opt/magma/secrets
```

Comment:
Can we use the `SECRETS_VOLUME` here rather than hardcoding the path as `/var/opt/magma/secrets`